sqlite/src/comdb2build.c

```diff
@@ -2237,17 +2237,18 @@ void comdb2getAnalyzeThreshold(Parse* pParse, Token *nm, Token *lnm)
                             (vdbeFuncArgFree)  &free, &stp);
 }
 
-int resolveTableName(struct SrcList_item *p, const char *zDB, char *tableName,
-                      size_t len)
+int resolveTableName(sqlite3 *db, struct SrcList_item *p, const char *zDB,
+                     char *tableName, size_t len)
 {
    struct sqlclntstate *clnt = get_sql_clnt();
    if ((zDB && (!strcasecmp(zDB, "main") || !strcasecmp(zDB, "temp"))))
    {
        snprintf(tableName, len, "%s", p->zName);
-   } else if (clnt && (clnt->user[0] != '\0') &&
-              !strchr(p->zName, '@') &&
-              strncasecmp(p->zName, "sqlite_", 7) &&
-              strncasecmp(p->zName, "comdb2", 6))
+   } else if (clnt &&
+              (clnt->user[0] != '\0') &&                /* authenticated */
```

Comment:
Didn't you change this to `current_user.user`?

Reply:
https://github.com/bloomberg/comdb2/pull/2296 hasn't merged yet.